tumblr_client.gemspec

```diff
@@ -4,14 +4,14 @@ require File.join(File.dirname(__FILE__), 'lib/tumblr/version')
 Gem::Specification.new do |gem|
   gem.add_dependency 'faraday', '~> 0.9.0'
   gem.add_dependency 'faraday_middleware', '~> 0.9'
-  gem.add_dependency 'json'
-  gem.add_dependency 'simple_oauth'
-  gem.add_dependency 'oauth'
-  gem.add_dependency 'mime-types'
-  gem.add_development_dependency 'rake'
-  gem.add_development_dependency 'rspec'
-  gem.add_development_dependency 'webmock'
-  gem.add_development_dependency 'simplecov'
+  gem.add_dependency 'json', '~> 1.8'
```

Comment:
Ah, that's helpful to know, thanks!

I also agree with your second point, but there currently isn't a `required_ruby_version` in the gemspec. I think in this PR I'll version off for Ruby 1.9.x, add the `required_ruby_version` to be `>= 1.9.2` (which is the lowest CI tests for), and then open a separate PR with lax version requirements. How does this sound? :)

And, thank you again for your comment! 🙇